### ISSUE TITLE
July Changelog Candidates

### ISSUE THREAD

**Issue by alexfornuto** [pantheon-systems/documentation#4789]
When entering product updates, please use the following format:

```
### Subject (i.e WP-CLI Upgrades)
Short description, like: WP-Cl has been upgraded to [1.5.0](https://make.wordpress.org/cli/2018/01/31/version-1-5-0-released/).
```

**Comment by greg-1-anderson** [comment]
### Drupal 8.7.3 
Update to Drupal 8.7.3. For more information, see https://www.drupal.org/project/drupal/releases/8.7.3

**Comment by greg-1-anderson** [comment]
### WordPress 5.2.2
Update to WordPress 5.2.2. For more information, see https://wordpress.org/news/2019/06/wordpress-5-2-2-maintenance-release/

**Comment by greg-1-anderson** [comment]
### Drupal 8.6.17
Update to Drupal 8.7.3. For more information, see https://www.drupal.org/project/drupal/releases/8.6.17

**Comment by ari-gold** [comment]
BUGS-2595

**Comment by greg-1-anderson** [comment]
### Drupal 8.7.4 
Update to Drupal 8.7.4. For more information, see https://www.drupal.org/project/drupal/releases/8.7.4

**Comment by alexfornuto** [comment]
Note to self: Move ^ to a new issue for August.

**Comment by ari-gold** [comment]
DNS recommendations update

**Comment by alexfornuto** [comment]
Please add new items to #4885

**Comment by alexfornuto** [comment]
Closed by #4853
